sotodlib/site_pipeline/update_det_match.py

```diff
@@ -163,15 +162,15 @@ def run_match_aman(runner: Runner, aman, detset, wafer_slot=None):
     stream_id = aman.det_info.stream_id[aman.det_info.detset == detset][0]
     sol_file = os.path.join(
         os.path.dirname(runner.detmap_cfg_path), 
```

Comment:
`.detmap_cfg_path` appears to be gone now.  Yes, I ran the code...

Reply:
Oops... thanks for catching, I was avoiding re-creating the full match archive to save time but clearly that was a mistake, and it did not cover the match-making parts.

This should be fixed and has been tested fully on SAT1 data with this pipeline-configs branch: https://github.com/simonsobs/site-pipeline-configs/pull/32